app.py

```
@@ -1,18 +1,30 @@
+import pandas as pd
```

Ruff (0.11.9)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)